code/modules/hivemind/objects.dm

```diff
@@ -0,0 +1,20 @@
+//Hivemind special objects stored here, like projectiles, wreckages or artifacts
+
+
+//toxic shot, turret's ability use it
+/obj/item/projectile/goo
+	name = "toxic goo"
+	icon = 'icons/obj/hivemind.dmi'
+	icon_state = "goo_proj"
+	damage = 10
+	damage_type = TOX
```

Comment:
this does nothing to robots afaik, perhaps a second type of projectile is needed that fires a laser or somesuch